scripts/get_latest_release.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Try to get the latest release
+url=$(curl -s https://api.github.com/repos/contiv/install/releases/latest  | grep -m 1 "browser_download_url")
+
+if [[ "$?" != "0" ]]; then
+  url=$(curl -s https://api.github.com/repos/contiv/install/releases  | grep -m 1 "browser_download_url")
+  if [[ "$?" != "0" ]]; then
+  	exit 1
+  fi
+fi
+release=$(echo $url | awk -F "/" '{print $8}')
```

Comment:
Can we do something like:

```bash
# (copied from StackOverflow and modified 😄)
python -c 'import json,sys;print json.load(sys.stdin)[0]["assets"][0]["browser_download_url"]')
```

in a function and pipe `curl` into that function to extract the URL?  Using grep + awk to split paths seems somewhat brittle to me.